pkg/util/hosts.go

```diff
@@ -1,52 +1,40 @@
 package util
 
 import (
-        "fmt"
-        "regexp"
-        "strings"
+	"fmt"
+	"regexp"
+	"strings"
 
-        corev1 "k8s.io/api/core/v1"
+	corev1 "k8s.io/api/core/v1"
 )
 
+var ansibleBlockRE = regexp.MustCompile(`(?s).*BEGIN ANSIBLE MANAGED BLOCK\n(.*)# END ANSIBLE MANAGED BLOCK.*`)
```

Comment:
I wonder if this can be a constant as well

Reply:
in the end this is going to be removed as we get the input from the controller deploy ansible vars where we don't have to use regex. This was only initially required during the POC when adding a generated hosts file as input from an existing external control plane.